apps/jfr-native-image-performance/worst_case_benchmark.hf.yaml

```diff
@@ -0,0 +1,39 @@
+name: jfr-hyperfoil
```

Comment:
I have two `hf.yaml` files because the test runs two separate benchmarks hitting separate endpoints. Yes I'll look into parameterize the benchmark instead (I saw your example in mechanical sympathy).

Reply:
ok the benchmark is now parameterized with `!param`. So now we only need one benchmark definition instead of two.